### PR TITLE
[SYCLomatic] Remove the using declaration if it is not std namespace

### PR DESCRIPTION
**Pull request by zhiweij1**
Signed-off-by: Jiang, Zhiwei <zhiwei.jiang@intel.com>

### CODE REVIEW

**Review by tomflinda — APPROVED** [review]
LGTM

File: clang/lib/DPCT/ASTTraversal.cpp [hunk 1]
```diff
@@ -13826,6 +13826,9 @@ void NamespaceRule::runRule(const MatchFinder::MatchResult &Result) {
         Repl += ";";
       }
       emplaceTransformation(new ReplaceText(Beg, Len, std::move(Repl)));
+      if (UD->getNameAsString() == "max" || UD->getNameAsString() == "min") {
```
Comment (zhimingwang36) [inline clang/lib/DPCT/ASTTraversal.cpp]: it should be better to remove using ::max instead of introduce dpct::max() function.

**Review by zhimingwang36 — CHANGES_REQUESTED** [review]
LGTM

**Review by zhimingwang36 — CHANGES_REQUESTED** [review]
LGTM

File: clang/lib/DPCT/ASTTraversal.cpp (hunk 1 above)
Comment (zhiweij1) [reply clang/lib/DPCT/ASTTraversal.cpp]: ok

**Review by zhimingwang36 — APPROVED** [review]
LGTM

**Pull request merged**